pkg/app/servers_test.go

```diff
@@ -116,8 +116,7 @@ func TestServeGRPC(t *testing.T) {
 		serveErr <- err
 	}()
 
-	conn, err := grpc.DialContext(clientCtx, addr,
-		grpc.WithBlock(),
```

Comment:
The old test had `WithBlock`, should we be using `WaitForReady` here?

Reply:
I considered it, but all that would achieve would be surfacing the error earlier, the next thing the test does is an RPC so any connection errors would show up there anyway